Sdk/Speckle.Connectors.Common/Caching/SendConversionCache.cs

```diff
@@ -7,9 +7,7 @@ namespace Speckle.Connectors.Common.Caching;
 ///<inheritdoc/>
 public class SendConversionCache : ISendConversionCache
 {
-  public SendConversionCache() { }
-
-  private Dictionary<(string applicationId, string projectId), ObjectReference> Cache { get; set; } = new(); // NOTE: as this dude's accessed from potentially more operations at the same time, it might be safer to bless him as a concurrent dictionary.
+  public Dictionary<(string applicationId, string projectId), ObjectReference> Cache { get; set; } = new(); // NOTE: as this dude's accessed from potentially more operations at the same time, it might be safer to bless him as a concurrent dictionary.
```

Comment:
Can we do `internal` members? with  an `InternalsVisibleToAttribute` for test projects?

Reply:
made internal with internals visible to